### ISSUE TITLE
Enhance a11y for `InfiniteScrollArea`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5411]
name: InfiniteScrollArea

### Description
5 accessibility violation was found in the `InfiniteScrollArea`.

#### [Components / InfiniteScrollArea / Root](https://yamada-ui.github.io/yamada-ui/?path=/story/components-infinitescrollarea--root)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / InfiniteScrollArea / Overflow](https://yamada-ui.github.io/yamada-ui/?path=/story/components-infinitescrollarea--overflow)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / InfiniteScrollArea / Orientation](https://yamada-ui.github.io/yamada-ui/?path=/story/components-infinitescrollarea--orientation)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / InfiniteScrollArea / Disabled](https://yamada-ui.github.io/yamada-ui/?path=/story/components-infinitescrollarea--disabled)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / InfiniteScrollArea / Reset](https://yamada-ui.github.io/yamada-ui/?path=/story/components-infinitescrollarea--reset)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

**Comment by hirotomoyamada** [comment]
Fixed #5655
